cmd/juju/application/store/charmadapter_test.go

```diff
@@ -312,7 +168,7 @@ func (s *resolveSuite) expectCharmResolutionCall(curl *charm.URL, out csparams.C
 
 func (s *resolveSuite) expectCharmResolutionCallWithAPIError(curl *charm.URL, out csparams.Channel, err error) {
```

Comment:
Replace "out csparams.Channel" as an arg. It's a charmstore specific old style channel.